app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -90,6 +92,11 @@ data class AppSettings(
         name = "sound_on_tap",
         defaultValue = DEFAULT_SOUND_ON_TAP.toString(),
     )
+    val spacebarSlide: Int,
+    @ColumnInfo(
+        name = "spacebar_slide",
```

Comment:
I have a feeling we should just name this `slide_enabled` and not `spacebar_slide`. Because in the future people might want to add a select slide, delete slide, etc.